includes/core-changes/aspnetcore/3.0/identity-ui-adddefaultui-overload-removed.md

```diff
@@ -0,0 +1,33 @@
+### Identity: AddDefaultUI method overload removed
+
+Starting with ASP.NET Core 3.0, the <xref:Microsoft.AspNetCore.Identity.IdentityBuilderUIExtensions.AddDefaultUI(Microsoft.AspNetCore.Identity.IdentityBuilder,Microsoft.AspNetCore.Identity.UI.UIFramework)?displayProperty=nameWithType> method overload no longer exists.
+
+#### Version introduced
+
+3.0
+
+#### Reason for change
+
+This change was a result of adoption of the static web assets feature.
+
+#### Recommended action
+
+Call <xref:Microsoft.AspNetCore.Identity.IdentityBuilderUIExtensions.AddDefaultUI(Microsoft.AspNetCore.Identity.IdentityBuilder)?displayProperty=nameWithType> instead of the overload. If you're using Bootstrap 3, also add the following line to a `<PropertyGroup>` element in your project file:
+
+`<IdentityUIFrameworkVersion>Bootstrap3</IdentityUIFrameworkVersion>`
```

Comment:
Wouldn't it render better like this?

```suggestion
```xml
<IdentityUIFrameworkVersion>Bootstrap3</IdentityUIFrameworkVersion>
```
```